listenbrainz_spark/stats/__init__.py

```diff
@@ -50,15 +50,23 @@ def replace_months(date, month):
     return date
 
 
-def adjust_months(date, months, shift_backwards=True):
+def offset_months(date, months, shift_backwards=True):
+    """Returns a datetime object :
+        - Decreased by number of months if shift_backwards is True
+        - Increased by number of months if shift backwards is False
+    """
```

Comment:
Can you please add the explanation about the Args which are supplied to the function as we do here [here](https://github.com/metabrainz/listenbrainz-server/blob/master/listenbrainz_spark/stats/user/utils.py#L28-L38). you can also combine the docstring for shift_backwards like 
`shift_backwards: If true decrease by the number of months, else increase.`